client/src/search.tsx

```diff
@@ -382,7 +382,7 @@ function InnerSearchNavigateWidget(props: InnerSearchNavigateWidgetProps) {
                 })}
               >
                 <a
-                  href={item.url}
+                  href={item.url.toLowerCase()}
```

Comment:
```suggestion
                  href={item.url.toLowerCase()}
```
What breaks without this `toLowerCase()` call?

Reply:
The links in the page title search would have uppercase letters (which currently leads to a redirect). Now that I think about it, it wouldn't be that hard to deal with it in `scripts/fix-urls.rs` instead by modifying `search-index.json`. I'll do that now.